pyproject.toml

```diff
@@ -110,7 +110,7 @@ dependencies = [
   "python-slugify<9",
   "pyyaml<7",
   "requests>=2.28,<3",
-  "sqlalchemy-cratedb>=0.41",
+  "sqlalchemy-cratedb>=0.42.0.dev2",
```

Comment:
This needs a release or be reverted. Otherwise, downstream users who use `uv` will need to add `--prerelease=allow` to install the package.